app/src/main/java/com/myperishableplanner/v21001/dto/Item.kt

```diff
@@ -1,3 +1,5 @@
 package com.myperishableplanner.v21001.dto
 
-data class Item()
+import java.util.*
+
+data class Item(var name: String, var category: String, var expirationDate: Date, var description: String)
```

Comment:
Item data class had an empty constructor which caused the project to not compile so I added parameters to the Item data class.

Reply:
Good.  Yes.  A data class requires parameters.  It can't have an empty constructor.